config/routes.rb

```diff
@@ -28,6 +28,12 @@
     resources  :categories, only: :index
     resources  :prospective_users, only: :create
     resources  :assessments, only: [:index, :show, :update, :create] do
+      collection do
+        resources :shared, controller: 'shared_assessments', only: [:show], param: :token do
```

Comment:
It's more a namespacing thing than anything else.  We're only getting one action out of those controllers, so it doesn't feel like a resource.

It's fine the way it is now, though.